components/next_steps_view/steps.ts

```diff
@@ -194,3 +196,15 @@ export const nextStepsNotFinished = createSelector(
         return !Steps.every(checkPref);
     },
 );
+
+export const userExistedForSomeTime = createSelector(
+    'userExistedForSomeTime',
+    (state: GlobalState) => getCurrentUser(state),
+    (currentUser): boolean => {
+        const today = moment(Date.now());
+        const created = moment(new Date(currentUser.create_at));
+        const d = today.diff(created, 'days');
+        const existedForSomeTime = d > NUMBER_OF_DAYS_AFTER_WHICH_TO_CONSIDER_AS_NON_NEW_USER;
+        return existedForSomeTime;
+    },
+);
```

Comment:
Should we move this to the users selectors here: https://github.com/mattermost/mattermost-webapp/blob/master/packages/mattermost-redux/src/selectors/entities/users.ts#L1